pkg/bbgo/config.go

```diff
@@ -478,6 +480,10 @@ func Load(configFile string, loadStrategies bool) (*Config, error) {
 	if err := yaml.Unmarshal(content, &config); err != nil {
 		return nil, err
 	}
+	config.ConfigFile = configFile
+	for _, session := range config.Sessions {
+		session.ConfigPath = config.ConfigFile
+	}
```

Comment:
Can you pull out the config things to another PR? if you would like this PR to get merged first.

I think the config saving stuff needs to be designed carefully.

Reply:
done